validator/js/webui/logo-blue.svg

```diff
@@ -1,17 +1,5 @@
-                <path d="M160.989904,31.0125833 L164.117187,31.0125833 C165.680409,31.0125833 166.805288,30.64425 167.492668,29.9050833 C168.180048,29.16675 168.523317,28.1350833 168.523317,26.8109167 C168.523317,26.2250833 168.434976,25.6909167 168.257452,25.2109167 C168.079087,24.7300833 167.809014,24.31675 167.448918,23.9709167 C167.087139,23.6259167 166.63113,23.3584167 166.080048,23.1709167 C165.529808,22.9834167 164.874399,22.8900833 164.117187,22.8900833 L160.989904,22.8900833 L160.989904,31.0125833 Z M160.989904,35.4959167 L160.989904,44.0584167 L154.983534,44.0584167 L154.983534,18.44175 L164.117187,18.44175 C165.941226,18.44175 167.507812,18.6559167 168.816947,19.0834167 C170.12524,19.5109167 171.203846,20.1000833 172.05024,20.8509167 C172.896635,21.6009167 173.522596,22.4859167 173.92476,23.5050833 C174.327764,24.5250833 174.528846,25.62675 174.528846,26.8109167 C174.528846,28.0884167 174.321875,29.2600833 173.907933,30.3275833 C173.492308,31.3934167 172.858774,32.3084167 172.00649,33.0700833 C171.153365,33.83175 170.072236,34.42675 168.763101,34.85425 C167.454808,35.2825833 165.905889,35.4959167 164.117187,35.4959167 L160.989904,35.4959167 Z" id="Fill-5"></path>
-                <path d="M40.6737019,27.2526667 L27.9676923,48.196 L25.6657692,48.196 L27.9416106,34.5493333 L20.8936538,34.5593333 C20.8936538,34.5593333 20.8271875,34.561 20.7952163,34.561 C20.1608413,34.561 19.6459375,34.051 19.6459375,33.4226667 C19.6459375,33.1526667 19.900024,32.696 19.900024,32.696 L32.5639663,11.7768333 L34.9045913,11.786 L32.5715385,25.4543333 L39.6556731,25.446 C39.6556731,25.446 39.7322356,25.4443333 39.7684135,25.4443333 C40.4027885,25.4443333 40.916851,25.9535 40.916851,26.5826667 C40.916851,26.8376667 40.8167308,27.0618333 40.6728606,27.2518333 L40.6737019,27.2526667 Z M30.288125,0.000166666667 C13.5604808,0.000166666667 -0.000336538462,13.4318333 -0.000336538462,30.0001667 C-0.000336538462,46.5685 13.5604808,60.0001667 30.288125,60.0001667 C47.0166106,60.0001667 60.5765865,46.5685 60.5765865,30.0001667 C60.5765865,13.4318333 47.0166106,0.000166666667 30.288125,0.000166666667 L30.288125,0.000166666667 Z" id="Fill-7"></path>
-            </g>
-        </g>
-    </g>
+<svg width="175" height="60" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Same as previous SVG, except I changed `fill` below and ran it through [SVGOMG](https://jakearchibald.github.io/svgomg/) for size

Reply:
> @honeybadgerdontcare Will this preserve the text "AMP" that is in the original SVG?

Yes